analysis_options.yaml

```diff
@@ -45,4 +45,14 @@ linter:
 
     join_return_with_assignment: false
 
-    prefer_const_literals_to_create_immutables: false
\ No newline at end of file
+    prefer_const_literals_to_create_immutables: false
+
+
+    ####################################################
+    #### Added to make things work for now. Must be ####
+    ############# removed in future PRs ################
+    ####################################################
+
+    depend_on_referenced_packages: false
```

Comment:
Does this need to be reverted to `true` after the merge? If so, add a comment stating this.